python-package/securexgboost/remote_server.py

```diff
@@ -634,13 +651,13 @@ def rpc_XGBoosterDumpModelEx(self, request, context):
         """
         try:
             if globals()["is_orchestrator"]:
-                length, sarr, status = self._synchronize(remote_api.XGBoosterDumpModelEx, request)
+                length, sarr, sig_proto, status = self._synchronize(remote_api.XGBoosterDumpModelEx, request)
```

Comment:
I suppose there should be a `sig_len` here too...

Reply:
Fixed